app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Yajra\Datatables\Datatables;
+use App\Models\User;
+use App\Http\Resources\User as UserResource;
+
+class UserController extends Controller
+{
+
+    public function index()
+    {
+        return view('admins/user');
+    }
+
+    public function getList()
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Formatting.DisallowMultipleStatements.SameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 20</br>Each PHP statement must be on a line by itself